hooks/use-render-appender-with-limit/index.ts

```diff
@@ -0,0 +1,24 @@
+import { useSelect } from '@wordpress/data';
+import {
+	store as blockEditorStore,
+	useBlockEditContext,
+	InnerBlocks,
+} from '@wordpress/block-editor';
+
+export function useRenderAppenderWithLimit(limit: number, buttonAppender = false) {
```

Comment:
API wise I would prefer if we just allowed to pass in the component that should be used as the appender as the second arg. That way it's more future proof and supports custom ones

Reply:
Done 🙂